test/trento/databases/database_test.exs

```diff
@@ -695,6 +760,421 @@ defmodule Trento.Databases.DatabaseTest do
     end
   end
 
+  describe "multi tenant support" do
+    test "should emit a DatabaseTenantsUpdated event when the database is registered" do
```

Comment:
I would say this test is abundantly done hehe
If you want to keep it for sake's of completeness is fine, but it is a copy/paste of the registration test

Reply:
Yeah I know it's just for completeness :D my ocd wants that kek